### ISSUE TITLE
Making it easier to contribute marker data

### ISSUE THREAD

**Issue by mathiasbynens** [tibiamaps/tibia-map-data#66]
In https://github.com/tibiamaps/tibia-map/issues/32, @LeeKunTibia brings up how much of a hassle it is to add markers by exploring in-game. There must be a better way!

I’ve just simplified the structure of the JSON data. For each floor, it’s now an array of objects, and each object represents a marker. See https://github.com/tibiamaps/tibia-map-data/blob/master/data/floor-07-markers.json for an example. Adding markers by editing the JSON is now easier than ever — however, submitting pull requests on GitHub for each change can still be painful.

Idea: set up a Google Docs spreadsheet where trusted contributors like @LeeKunTibia and @Hunter-of-Dragoes can enter the following:

1. a `tibiamaps.io/map` URL

    example: `https://tibiamaps.io/map#32359,32126,7:3`

2. the icon type (see the “short name” column on https://tibiamaps.io/guides/map-file-format#map-marker-data)

    example: `star`

3. optionally, a description

    example: `Levitate spot`

I could then “sync” the spreadsheet every two weeks or so with this repository, adding the new markers.

This would make it significantly easier to suggest new markers, without having to necessarily open the game client and upload any files.

@LeeKunTibia, @Hunter-of-Dragoes, what do you think?

**Comment by mathiasbynens** [comment]
Pinged Reddit as well: https://www.reddit.com/r/TibiaMMO/comments/98twon/if_you_could_suggest_markers_to_tibiamapsio_by/

**Comment by mathiasbynens** [comment]
I know Lee was mainly talking about something else in the other issue, i.e. adding a separate set of markers that would only show up on https://tibiamaps.io/map. That’s why I created this issue to discuss this other idea :)

**Comment by LeeKunTibia** [comment]
Indeed, what I was referring to was markers that would only show up on the website. Nevertheless, this change seems very good. I have personally not used the TibiaMaps markers since there is no easy way to merge them with mines, but this will make things much easier. 

I suppose the original idea could be implemented very similarly, with an extra set of markers exclusive of the website (i.e. _Tanjis Boss Room_). The most time consuming part is developing the website feature, I guess.

**Comment by mathiasbynens** [comment]
Any new areas since the Feyrist update do not currently have markers. Suggestions for those are more than welcome!

> I'm also adding markers for the new Asura area, the Secret Library and the Falcon Bastion.

Yay!

**Comment by mathiasbynens** [comment]
> I'm not sure however if I should also add all NPCs and Quest impotant points, as it can get too messy there […]

It’s better to have too many markers than too few. People can easily delete individual markers, or download the maps without markers to keep their own markers (like you).
